src/queue.js

```diff
@@ -219,4 +219,6 @@ class Queue extends EventEmitter {
   }
 }
 
+Queue.Job = null; // Make TypeScript compiler happy. Needed because Job is an abstract class.
```

Comment:
OK. Well seeing as this API is being added to the Queue object purely to support TypeScript, I'm happy for it to export the required `Job` object from  within `Queue` with a capital J.

Lets go with `Queue.Job = Job` as you first suggested. I will document the API so clients know what it is for.